### DISCUSSION TITLE
How to Recognize Anime that doesn't have the title in the episode name

### DISCUSSION THREAD

**Discussion by Adaptz** [erengy/taiga#1149] [Questions]
I have a download where the episodes only have the number and the name of the episode. Can I not just set the folder as the anime I'm watching and it just go by episode number? If i can how do i do that?

**Comment by Adaptz** [comment]
I've set the folder under information for the anime I'm watching as the correct folder, but it doesn't seem to do anything.

https://i.imgur.com/YWMP5sh.png

I've also just set the episode name to have the anime name in it in hopes I could get anything to work, but the auto recognized anime it gave me are all the specials and movies rather than the main series of the anime.

https://i.imgur.com/NLuBKpM.png

**Reply by erengy** [reply]
> I've set the folder under information for the anime I'm watching as the correct folder, but it doesn't seem to do anything.

Anime folders don't affect recognition, they're just where Taiga looks first while scanning available episodes. I'll try to make this more obvious in future releases.

**Comment by erengy** [comment]
Taiga can read the anime title from the folder and the episode number from the files. The following paths work fine, for instance:

- `C:\Anime\Violet Evergarden\01 - 'I Love You' and Auto Memory Dolls.mkv`
- `C:\Anime\Violet Evergarden\S01E01 - 'I Love You' and Auto Memory Dolls.mkv`
- `C:\Anime\Violet Evergarden [1080p Dual Audio BDRip 10 bits]\01 - 'I Love You' and Auto Memory Dolls.mkv`

Make sure the folder name doesn't have too much clutter (e.g. `S01+Movies+OVA 1080p Dual Audio BDRip 10 bits`) and the episode number is separated properly (e.g. `01 - ` instead of `01-`).

**Reply by Adaptz** [reply]
I figured out the issue it doesn't recognize the episode when their together like "S01E01" when I changed it to "S01 E01" it worked perfectly fine.

**Reply by Adaptz** [reply]
Thank you for the help, I know this isn't the proper place for this but, if I could request that in a future update you change it to recognize the episode when it's not separated that would be amazing. Luckily I only had to change 14 episode names, but that'd really suck for a longer anime like one piece.

**Reply by erengy** [reply]
Taiga can already recognize `S01E01` though, see the second example above. I think yours might have had a different issue. Could you post the paths (including the filename) for what did and did not get recognized?

**Reply by Adaptz** [reply]
@erengy 
Yeah sure, the original path that didn't work is:

`C:\Users\*****\Videos\Anime\Violet Evergarden\Violet Evergarden [Season 1 + OVA]\S01E01-I Love You' and Auto Memory Dolls [F03E1F7A]`

https://i.imgur.com/Rl6SBvG.png



The path that does work is:

`C:\Users\*****\Videos\Anime\Violet Evergarden\Violet Evergarden [Season 1 + OVA]\S01 E01-I Love You' and Auto Memory Dolls [F03E1F7A]`

This one perfectly recognizes it without it even asking for suggested titles.

I'm thinking the issue that's occurring here is that there's no spaces after S01E01 just the "-" since the following file path works perfectly fine:

`"C:\Users\*****\Videos\Anime\Violet Evergarden\Violet Evergarden [Season 1 + OVA]\S01E01 -I Love You' and Auto Memory Dolls [F03E1F7A].mkv"`

**Reply by erengy** [reply]
> I'm thinking the issue that's occurring here is that there's no spaces after S01E01 just the "-"

Exactly. `S01E01 - I Love` instead of `S01E01-I Love` should work. See #1071 and #1069. It's a common issue, but cannot be solved at the moment, unfortunately.
